cfme/automate/dialog_element.py

```diff
@@ -0,0 +1,196 @@
+from navmazing import NavigateToAttribute
+from widgetastic.widget import Checkbox, Image, Text
+from widgetastic_patternfly import Button, Input, BootstrapSelect
+from widgetastic_manageiq import ManageIQTree, Table, TextInput
+from widgetastic.xpath import quote
+
+from cfme.fixtures import pytest_selenium as sel
```

Comment:
This has to go. Use the `appliance.browser.widgetastic` or `<widgetasticwidget>.browser` for raw queries.